### PR TITLE
updating the guidelines

### PR DESCRIPTION
**Pull request by kalexand-rh**
@vikram-redhat, PTAL

### CODE REVIEW

File: contributing_to_docs/contributing.adoc [hunk 1]
```diff
@@ -11,36 +11,35 @@ toc::[]
...
-If there are any further changes, updates, or corrections required, we will
-inform you in the PR. When the PR has been reviewed and all changes have been
-accepted by our documentation team, your PR will be merged.
+The
+https://github.com/orgs/openshift/teams/team-documentation[documentation team]
+reviews the PR and arranges further review by the development and quality
+assurance teams, as required.
+If the PR needs changes, updates, or corrections required, we will let you know
+in the PR. We might request that you make the changes or let you know that we
+incorporate your content in a different PR. When the PR has been reviewed, all
```
Comment (vikram-redhat) [inline contributing_to_docs/contributing.adoc]: typo

File: contributing_to_docs/contributing.adoc [hunk 2]
```diff
@@ -51,25 +50,31 @@ topic are stored.
 /topic_dir/topicN.adoc
```
Comment (vikram-redhat) [inline contributing_to_docs/contributing.adoc]: change that to assemblyN.adoc?

File: contributing_to_docs/contributing.adoc [hunk 3]
```diff
@@ -51,25 +50,31 @@ topic are stored.
...
 /topic_dir/subtopic_dir1/topic1.adoc
```
Comment (vikram-redhat) [inline contributing_to_docs/contributing.adoc]: Same, sub-assembly1.adoc?

File: contributing_to_docs/contributing.adoc [hunk 4]
```diff
@@ -51,25 +50,31 @@ topic are stored.
...
 ...
 /topic_dir2
 ----
 
 == Version management
-Most of the content applies to all four OpenShift products: Origin, Online, Dedicated and
-Enterprise (some versions of Enterprise), so approximately 80% of the content is reused. In many cases, this
-means that individual topics may need to include or exclude individual
-paragraphs with respect to a specific OpenShift product. While it is _possible_
+Most of the content applies to all four OpenShift products: ODK, OpenShift
```
Comment (vikram-redhat) [inline contributing_to_docs/contributing.adoc]: OKD

File: contributing_to_docs/contributing.adoc [hunk 5]
```diff
@@ -93,41 +98,41 @@ Here is an example of this concept in use:
 This first line is unconditionalized, and will appear for all versions.
 
 \ifdef::openshift-online[]
-This line will only appear for OpenShift Online.
+This line will only appear for ODK.
```
Comment (vikram-redhat) [inline contributing_to_docs/contributing.adoc]: This should stay as OpenShift Online?

File: contributing_to_docs/contributing.adoc [hunk 6]
```diff
@@ -93,41 +98,41 @@ Here is an example of this concept in use:
...
 endif::[]
 
 \ifdef::openshift-enterprise[]
-This line will only appear for OpenShift Enterprise.
+This line will only appear for OpenShift Container Platform.
 endif::[]
 
 \ifdef::openshift-origin,openshift-enterprise[]
-This line will appear for OpenShift Origin and Enterprise, but not for OpenShift Online or OpenShift Dedicated.
+This line will appear for ODK and OpenShift Container Platform, but not for OpenShift Online or OpenShift Dedicated.
```
Comment (vikram-redhat) [inline contributing_to_docs/contributing.adoc]: OKD

File: contributing_to_docs/contributing.adoc [hunk 7]
```diff
@@ -140,21 +145,25 @@ documentation after 6 hours.
 ====
 All OpenShift content development occurs on the `master`, or *working* branch.
```
Comment (vikram-redhat) [inline contributing_to_docs/contributing.adoc]: This is technically the case from when 4.0 launches and for current 3.11. I don't know whether to keep it here till then, if it is wrong. I vote to keep it as most folks who are contributing know the difference.

File: contributing_to_docs/contributing.adoc [hunk 8]
```diff
@@ -140,21 +145,25 @@ documentation after 6 hours.
...
+specify it in your PR.
 ====
 
-[[how-it-all-comes-together]]
-== How it all comes together
-The documentation build system reads the `&#95;distro&#95;map.yml` to figure out which branches to build and then the `&#95;topic&#95;map.yml` metadata file for each of the branches
+== Adding files to the collection
+After you create files, you must add them to the `&#95;topic&#95;map.yml` so
+that the build system can render them.
+The documentation build system reads the `&#95;distro&#95;map.yml` to determine
```
Comment (vikram-redhat) [inline contributing_to_docs/contributing.adoc]: Could you add "reads the _distro_map.yml from the master branch"

File: contributing_to_docs/contributing.adoc (hunk 7 above)
Comment (kalexand-rh) [reply contributing_to_docs/contributing.adoc]: Right. I also don't want to assume that we're going to remember to do a good scrub on these files before we GA.

File: contributing_to_docs/create_or_edit_content.adoc [hunk 9]
```diff
@@ -9,21 +9,29 @@
...
 with the remote repository.
 
+[NOTE]
+====
+Because most changes in this repository must be committed to the `master`
+branch, the following process always uses `master` as the name of the source
+branch. If you need to use another branch as the source for your change, make
+sure that you consistently use that branch name instead of `master`
+====
+
```
Comment (vikram-redhat) [inline contributing_to_docs/create_or_edit_content.adoc]: This note is not relevant for 4.0 but makes sense once 4.0 is launched and we switch master to it. I am in favor of keeping it.

File: contributing_to_docs/doc_guidelines.adoc [hunk 10]
```diff
@@ -50,9 +47,11 @@ Every module should be placed in the modules folder and should contain the follo
 = Module title                                                     <3>
 ----
 
-<1> List of assemblies in which this module is included
+<1> List of assemblies in which this module is included.
 <2> A module anchor with {context} that must match the module's file name.
-<3> Human readable title
+<3> Human readable title. To ensure consistency in the results of the
+leveloffset values in include statements, you must use a level one heading
+( = ) for the module title."
```
Comment (vikram-redhat) [inline contributing_to_docs/doc_guidelines.adoc]: "" typo

File: contributing_to_docs/tools_and_setup.adoc [hunk 11]
```diff
@@ -82,25 +80,12 @@ live content editing on a Fedora Linux system.
 5. Install _make_ with `yum install make`
 6. Install the _ascii_binder_ gem with `gem install ascii_binder`
 
-NOTE: If you already have AsciiBinder installed, you may be due for an update. These directions assume that you are using AsciiBinder 0.0.8 or newer. To check and update if necessary, simply run `gem update ascii_binder`. Note that you may need root permissions.
+NOTE: If you already have AsciiBinder installed, you might be due for an update.
+These directions assume that you are using AsciiBinder 0.0.8 or newer. To check
```
Comment (vikram-redhat) [inline contributing_to_docs/tools_and_setup.adoc]: Update this to 0.1.15

**Comment by kalexand-rh** [comment]
I've double-checked, and all of the changes are in. Merging.

**Pull request merged**